### PR TITLE
Add NeoChat

### PR DESCRIPTION
**Pull request by CarlSchwan**

### CODE REVIEW

File: src/open/clients/NeoChat.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import {Maturity, Platform, LinkKind, FlathubLink, style} from "../types.js";
+
+export class NeoChat {
+    get id() { return "neochat"; }
+    get name() { return "NeoChat"; }
+    get icon() { return "images/client-icons/quaternion.svg"; }
+    get author() { return "Tobias Fella and Carl Schwan"; }
+    get homepage() { return "https://apps.kde.org/neochat/"; }
+    get platforms() { return [Platform.Linux, Platform.Windows, Platform.macOS, Platform.Android]; }
+    get description() { return 'NeoChat is a Matrix convergent client fully integrated inside Plasma Mobile and Plasma Desktop, but it also works on many other platforms.'; }
```
Comment (nicolasfella) [inline src/open/clients/NeoChat.js:27]:
```suggestion
    get description() { return 'NeoChat is a convergent, cross-platform Matrix client.'; }
```

**Comment by CarlSchwan** [comment]
Friendly ping @bwindels :)

**Comment by CarlSchwan** [comment]
Ping?

File: src/open/clients/NeoChat.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+import {Maturity, Platform, LinkKind, FlathubLink, style} from "../types.js";
+
+export class NeoChat {
+    get id() { return "neochat"; }
+    get name() { return "NeoChat"; }
+    get icon() { return "images/client-icons/org.kde.neochat.svg"; }
+    get author() { return "Tobias Fella and Carl Schwan"; }
+    get homepage() { return "https://apps.kde.org/neochat/"; }
+    get platforms() { return [Platform.Linux, Platform.Windows, Platform.macOS, Platform.Android]; }
```
Comment (thibaultamartin) [inline src/open/clients/NeoChat.js:26]: After talking with the maintainers, it seems there is no official release on other platforms than Linux. It probably doesn't make sense to advertise NeoChat on other platforms for now

**Review by thibaultamartin — CHANGES_REQUESTED** [review]

**Review by thibaultamartin — APPROVED** [review]

**Comment by thibaultamartin** [comment]
Thanks Carl!

**Pull request merged**